ios-target/View/Home/HomeViewController.swift

```diff
@@ -25,12 +25,18 @@ class HomeViewController: UIViewController {
     viewModel.loadTargetPoints()
     mapView.delegate = self
   }
-  
+
   // MARK: - Actions
-  
-  @IBAction func tapOnAddTarget(_ sender: Any) {
-    //TODO:
-//    target.showTargetForm()
+  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+    if segue.destination is TargetFormViewController,
+      let viewController = segue.destination as? TargetFormViewController,
+      let locationLatitude = viewModel.locationLatitude,
+      let locationLongitude = viewModel.locationLongitude
```

Comment:
default latitude and longitude to 0 in the view model and thus avoid having to check existence